frontend/www/js/omegaup/components/teamsgroup/Upload.test.ts

```diff
@@ -1,24 +1,48 @@
 import { mount, shallowMount } from '@vue/test-utils';
+import { types } from '../../api_types';
 
 import T from '../../lang';
 
 import teamsgroup_Upload from './Upload.vue';
 
 describe('Upload.vue', () => {
-  it('Should handle upload teams view', () => {
+  it('Should handle upload teams view with identities', async () => {
     const wrapper = shallowMount(teamsgroup_Upload, {
       propsData: {
-        groupAlias: 'Hello',
+        searchResultUsers: [] as types.ListItem[],
       },
     });
 
     expect(wrapper.text()).toContain(T.groupsUploadCsvFile);
+
+    const identities = [
+      {
+        username: 'team_user_1',
+        name: 'user 1',
+        country_id: 'MX',
+        state_id: 'QUE',
+        gender: 'decline',
+        school_name: 'First School',
+      },
+    ] as types.Identity[];
```

Comment:
```suggestion
    const identities: types.Identity[] = [
      {
        username: 'team_user_1',
        name: 'user 1',
        country_id: 'MX',
        state_id: 'QUE',
        gender: 'decline',
        school_name: 'First School',
      },
    ];
```

siempre que `as` se pueda mover a la declaración, hay que hacerlo. `as` sólo se vale para:

* literales súper simples como `[] as Foo` o `null as Foo | null`.
* `JSON.parse(...) as Foo` (o conversiones de otras expresiones no-literales)

cualquier arreglo u objeto más grande, hay que moverlo a declaración de variable.

Reply:
Listo.